src/bubble/bubbles.clj

```diff
@@ -40,12 +41,16 @@
     (println (bubble-members param-id))
     (views/base-view
      [[:h1 (:bubbles/name bubble)]
+      [:h2 (str "Join bubble here: localhost:3000/bubble/" param-id "/join")]
       [:a {:href "/"} "Home"]
       [:ul
        (map (fn [user]
     ;; TODO: show names members once we have name capture
               [:li (:users/phone user)])
-            (bubble-members param-id))]])))
+            (bubble-members param-id))]
+      [:form {:action param-id :method "post"}
+       [:input {:placeholder "Your phone" :name "userphone"}]
+       [:button () "join bubble"]]])))
```

Comment:
I don't think this form belongs on this page, given that you already have to be logged in and a member of the bubble to view it.

Reply:
Removed this!